Utilities/MCStepLogger/include/MCStepLogger/ROOTManager.h

```diff
@@ -0,0 +1,163 @@
+/* Wrapping some functionality of handling ROOT files
```

Comment:
Please include the copyright statement (in all file). @dberzano : This shows that the codechecker is broken as it passes although the copyright statement is not there. See also https://alice.its.cern.ch/jira/projects/O2/issues/O2-283